packages/devtools_app/lib/src/screens/inspector/layout_explorer/flex/flex.dart

```diff
@@ -201,22 +206,22 @@ class _FlexLayoutExplorerWidgetState extends LayoutExplorerWidgetState<
                 ),
               )
           ],
-          onChanged: (Object newSelection) async {
+          onChanged: (Object? newSelection) async {
             // newSelection is an object instead of type here because
             // the type is dependent on the `axis` parameter
             // if the axis is the main axis the type should be [MainAxisAlignment]
             // if the axis is the cross axis the type should be [CrossAxisAlignment]
             FlexLayoutProperties changedProperties;
             if (axis == direction) {
-              changedProperties =
-                  properties.copyWith(mainAxisAlignment: newSelection);
+              changedProperties = propertiesLocal.copyWith(
+                  mainAxisAlignment: newSelection as MainAxisAlignment?);
```

Comment:
more cases with as checks that should be removed or replaced with `!` operators.